src/main/resources/knowledge_environment.graphqls

```diff
@@ -16,6 +16,13 @@ type Query {
   getTotalParticipantFilesCount(redcapId: String): ParticipantRepoDataTypeInformation
   getTissueTypeSummaryData: [ParticipantTissueTypeSummary]
   getAtlasSummaryRows: AtlasRepoSummaryResult
+  getAtlasMessages: [AtlasMessages]
+}
+
+type AtlasMessages {
+  message: String
+  startDate: String

```

Comment:
We are going to want/need to also return the id of the message so that we can keep track of which ones have been viewed in the applicaiton.